src/scripts/main/public/html/task-list.html

```diff
@@ -20,66 +20,47 @@ <h1>TaskList</h1>
       <nav class="sidebar">
       </nav>
       <!--This container includes the header and the data content for format-->        
-        <body>
-          <div class="to-do-list">
-            <div class="div">
-              <div class = "tasks-container">
-                <div class="overlap">
-                  <label class="group">
-                    <input type="checkbox" id="task-checkbox" class="rectangle-checkbox">
-                    <div class="text-wrapper">Add new task-list to get started</div>
-                  </label>
-                  <button class="delete-btn">&#x2716;</button>
-                </div>
-              </div>  
-            </div>
-
-            <div class="modal" id="modal">
-              <div class="modal-content">
-                  <span class="close" id="closeModal">&times;</span>
-                  <!-- This div will be replaced with the content loaded from the external HTML file -->
-                  <div id="popupContent"></div>
-              </div>
-            </div>
-
-            <!--pop-up page for add tasklist-->
-            <div class="union-wrapper">
-                <button id="openModal" class = "button-style">
-                  <img class="union" src="../img/union.svg" />
+        <div class="to-do-list">
+          
+          <div class="div">
+            <div class = "tasks-container">
+              <div class="overlap">
+                <label class="group">
+                  <input type="checkbox" id="task-checkbox" class="rectangle-checkbox">
```

Comment:
IDs should be camelCase

Reply:
Thank you for the feedback,
I just renamed the ID to ensure it follows the camelCase style.
Please let me know if there's anything else I need to fix.